Form-Controls/index.html

```diff
@@ -1,27 +1,49 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="utf-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <title>My form exercise</title>
-    <meta name="description" content="" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-  </head>
-  <body>
-    <header>
-      <h1>Product Pick</h1>
-    </header>
-    <main>
-      <form>
-        <!-- write your html here-->
-        <!-- 
-          try writing out the requirements first as comments
-        this will also help you fill in your PR message later-->
-      </form>
-    </main>
-    <footer>
-      <!-- change to your name-->
-      <h2>By HOMEWORK SOLUTION</h2>
-    </footer>
-  </body>
+<head>
+  <meta charset="UTF-8">
+  <title>T-Shirt Order Form</title>
+</head>
+<body>
+
+  <h1>Order Your T-Shirt</h1>
+
+  <form>
+    <!-- Name -->
+    <label for="name">Full Name:</label>
+    <input type="text" id="name" name="name" required minlength="2" />
+
+    <!-- Email -->
+    <label for="email">Email Address:</label>
+    <input type="email" id="email" name="email" required />
+
+    <!-- Colour -->
+    <p>Select a colour:</p>
+    <input type="radio" id="red" name="color" value="Red" required />
+    <label for="red">Red</label>
+
+    <input type="radio" id="blue" name="color" value="Blue" />
+    <label for="blue">Blue</label>
+
+    <input type="radio" id="green" name="color" value="Green" />
+    <label for="green">Green</label>
+
+    <!-- Size -->
+    <label for="size">Choose a size:</label>
```

Comment:
Visually, it's difficult to see the size selection when it's squished right next to the color radio buttons.
![Screenshot 2025-05-17 at 8 57 10 PM](https://github.com/user-attachments/assets/0b20abe6-4748-4c19-b8d7-788a97111137)
Can you think of a way to make the UI look better and easier to read for the user?